### PR TITLE
Pim 5900: Unify forms

### PR DESCRIPTION
**Pull request by pierallard**
# One form for rule them all

screenshot from 2016-08-04 11 57 38

The goal is to have only one design for every form in the application:
- labels under inputs
- same font size everywhere
- same input width
- etc...

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | n
| Added Behats                      | n
| Changelog updated                 | n
| Review and 2 GTM                  | TODO
| Micro Demo to the PO (Story only) | n
| Migration script                  | n
| Tech Doc                          | n

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less [hunk 1]
```diff
@@ -110,49 +110,66 @@ a:focus {
...
         content:":";
       }
     }
+
     .controls {
-      margin-left:170px;
-      .control-label {
-        width: auto;
+      margin-left: 0;
+      input[type="text"], textarea, .select2-container {
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less]: please always add a blank line before selector

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 2]
```diff
@@ -1689,7 +1689,7 @@ h3.tab-header {
 
 .attributes .input-like, [data-type="pim-filter-product-category"] .filter-input .input-like {
     padding: 2px 2px 2px 0;
-    margin: 0 0 10px 5px;
+    margin: 0 0 10px;
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: we need to remove that, completelly

**Comment by juliensnz** [comment]
GTM after comment fix

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 3]
```diff
@@ -1648,8 +1648,9 @@ h3.tab-header {
 
 .input-like {
   padding: 0 3px 10px 3px;
-  width: 300px;
+  width: 500px;
```
Comment (loicguillois) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: should be a variable IMHO

File: src/Pim/Bundle/UIBundle/Resources/public/css/select2.less [hunk 4]
```diff
@@ -259,7 +259,7 @@ a.select2-choice {
 }
 
 .form-horizontal .select2-container {
-  width: 300px ;
+  width: 500px ;
```
Comment (loicguillois) [inline src/Pim/Bundle/UIBundle/Resources/public/css/select2.less]: should be a variable IMHO

**Comment by loicguillois** [comment]
GTM because legacy but in the future we should forbid this kind of CSS structure

If curious, here is an introduction : http://www.alsacreations.com/article/lire/1641-bonnes-pratiques-en-css-bem-et-oocss.html

**Pull request merged**